python/lsst/ts/nightreport/main.py

```diff
@@ -0,0 +1,54 @@
+import fastapi
+import fastapi.responses
+import starlette.requests
+
+from . import shared_state
+from .routers import (
+    add_nightreport,
+    delete_nightreport,
+    edit_nightreport,
+    find_nightreports,
+    get_configuration,
+    get_nightreport,
+)
+
+app = fastapi.FastAPI()
+
+subapp = fastapi.FastAPI(
+    title="Night report service",
+    description="A REST web service to create and manage night reports for the Vera C. Rubin Observatory.",
+)
+app.mount("/nightreport", subapp)
+
+subapp.include_router(add_nightreport.router)
+subapp.include_router(delete_nightreport.router)
+subapp.include_router(edit_nightreport.router)
+subapp.include_router(find_nightreports.router)
+subapp.include_router(get_configuration.router)
+subapp.include_router(get_nightreport.router)
+
+
+@subapp.get("/", response_class=fastapi.responses.HTMLResponse)
+async def root(request: starlette.requests.Request) -> str:
```

Comment:
Please add a doc string explaining what the function does and what the parameters represent here and everywhere else.